dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dependencies;
+
+import java.io.File;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RepositoryUtilityTest {
+
+  @Test
+  public void testFindLocalRepository() {
+    File local = RepositoryUtility.findLocalRepository();
+    Assert.assertTrue(local.exists());
```

Comment:
Is there anything you can assert about it other than this?

Reply:
canRead and canWrite I suppose. Any other ideas?

Reply:
Is there any metadata that lets you know it didn't just pick / or /tmp?

Does the repo system add a dot-file? Can we assert that it's either empty or its basename is ".m2"?

Reply:
I don't think we can assert either of those. If we don't find the local repo this just creates a temporary directory. I don't think the repo system puts anything there until it starts loading dependencies.